index.js

```diff
@@ -8,9 +8,18 @@ var tagParser = require('./app/tag_parser');
 
 
 var xejs = function(file, options, parentPath) {
-    var dirname = file;
-    if (parentPath) dirname = path.join(parentPath, "../", file);
-    var content = fs.readFileSync(dirname, 'utf-8');
+    var content;
+
+    if(options.stringRender) {
+        // Render contents from file var
+        content = file;
```

Comment:
I think not applying the string render to includes would be the best thing to do. what do you think?